app/models/person.rb

```diff
@@ -60,6 +64,12 @@ def agent_projects
     Project.where("primary_agent_id = ? OR secondary_agent_id = ?", id, id)
   end
 
+  def notification_source
```

Comment:
In creating this method I am just following how other user attributes are persisted and accessed here, but it is not a very nice way of doing things. For models that are nested inside one another, accepts_nested_attributes_for is the better way to do this. But as noted above, I think the two should be combined. So I didn't think it was worthwhile to refactor this class to more gracefully accept nested attributes at this time.